tests/pkg_u8g2/Makefile

```diff
@@ -30,6 +30,8 @@ ifeq ($(TEST_OUTPUT),1)
 endif
 
 ifeq ($(TEST_OUTPUT),2)
+# SDL requires more stack
+  CFLAGS += -DTHREAD_STACKSIZE_MAIN=64*1024
```

Comment:
I don't remember if I used 64Kb just cause or if I played around with the value a bit, did you try it out?

Reply:
done 32k wasn't enough

Reply:
https://github.com/rofl0r/df-libgraphics/blob/nogl/dist/sdl-stacksize.patch < dwarf-fortress has a 256kB patch for musl+sdl  (but they also have much more on display)

Reply:
Lets leave the minimum that works in the Makefile, but how about adding a hint that more could be required in the doc.txt?